src/libsystemd/sd-netlink/test-netlink-util.c

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <net/if.h>
+#include <unistd.h>
+
+#include "netlink-util.h"
+#include "strv.h"
+#include "tests.h"
+
+static void test_rtnl_set_link_name(sd_netlink *rtnl, int ifindex) {
```

Comment:
Could you move this test to test-netlink.c? I do not think it is necessary to introduce a new test executable.